### PR TITLE
Merge 21.07 into master 20210730

### PR DESCRIPTION
**Pull request by nollymar**

### CODE REVIEW

File: .github/workflows/core-cicd-tests.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
     name: Unit Tests
     runs-on: ubuntu-latest
     env:
-      DOT_CICD_BRANCH: master
+      DOT_CICD_BRANCH: release-21.07
```
Comment (erickgonzalez) [inline .github/workflows/core-cicd-tests.yml:14]: use master

File: .github/workflows/vulnerability-scan.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ jobs:
       matrix:
         scan: [ api, baseline, full ]
     env:
-      DOT_CICD_BRANCH: master
+      DOT_CICD_BRANCH: release-21.07
```
Comment (erickgonzalez) [inline .github/workflows/vulnerability-scan.yml:13]: same

**Review by erickgonzalez — APPROVED** [review]

**Review by freddyucv — APPROVED** [review]

**Review by jdotcms — APPROVED** [review]

**Review by dsilvam — APPROVED** [review]

File: .github/workflows/vulnerability-scan.yml (hunk 2 above)
Comment (nollymar) [reply .github/workflows/vulnerability-scan.yml]: Done

File: .github/workflows/core-cicd-tests.yml (hunk 1 above)
Comment (nollymar) [reply .github/workflows/core-cicd-tests.yml]: Done

**Pull request merged**